src/components/entities/filters/typeFilter.js

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
```

Comment:
how is this filter populated?

Reply:
> how is this filter populated?

Dynamically with the data retrieved without the cloned values

Reply:
so this is populated from the entities data.
ideally you should retrieve it from the api:
`/v2/types`

so if we had a resource type `entity type`, that could store the endpoint to retrieve the types.
the query returns a list as follows: 
```
[
  {
    "type": "Car",
    "attrs": {
      "speed": {
        "types": [
          "Number"
        ]
      },
      "fuel": {
        "types": [
          "gasoline",
          "diesel"
        ]
      },
      "temperature": {
        "types": [
          "urn:phenomenum:temperature"
        ]
      }
    },
    "count": 12
  },
  {
    "type": "Room",
    "attrs": {
      "pressure": {
        "types": [
          "Number"
        ]
      },
      "humidity": {
        "types": [
          "percentage"
        ]
      },
      "temperature": {
        "types": [
          "urn:phenomenum:temperature"
        ]
      }
    },
    "count": 7
  }
]
```
the name of the type is in the attribute `type`.

Reply:
it's also fine to not handle this in this pr. the point though is that this is they way to know all entity types (also the ones for which there is no entity at the time being)